api-v2.rst

```diff
@@ -0,0 +1,44 @@
+.. post:: July 26, 2018
+   :tags: api, feature, reference
+   :author: David
+   :location: SAN
+
+Read the Docs Public API
+========================
+
+Recently, we revamped Read the Docs' public API.
+Previously, our latest API (v2) was used by our build processes
+but not heavily used by outside users.
+
+As part of this process, we put effort into making sure the API is easy to use
+to access Read the Docs *projects*, *builds*, and *versions*
+and that the `documentation`_ is up-to-date.
```

Comment:
Doesn't worth to mention what are those changes? Like adding the `project__slug` filter and `active`, for example?

Reply:
I added a note about filtering specifically. Considering there wasn't any documentation before, I think mentioning `project__slug` specifically isn't necessary. Do you think that's ok?

Reply:
The note is good.

I'd say that it's not _necessary_. It would depend on how technical we want to be in the post. It may add more noise than clarification, though.